apps/src/templates/verificationPages/WebLabNetworkCheck.jsx

```diff
@@ -252,9 +248,11 @@ class WebLabNetworkCheck extends Component {
             When checking network support, try to meet as many of these criteria
             as possible:
           </p>
-          <li>Use your school's internet connection</li>
-          <li>Use a student computer</li>
-          <li>Log into the computer with a student account</li>
+          <ul>
```

Comment:
Adding the `<ul>` parent tag here fixes the identation of the child bullet points for the better. You can see the difference in one of the four screenshots in the "Testing" section that looks different than the others.